doc/classes/DisplayServer.xml

```diff
@@ -136,6 +136,7 @@
 			<param index="6" name="callback" type="Callable" />
 			<description>
 				Displays OS native dialog for selecting files or directories in the file system.
+				Each filter string in the [param filters] array is to be formatted similar to this: [code]*.txt;Text Files (*.txt)[/code]
```

Comment:
It could reference `FileDialog.filters`, this is done in a couple other similar cases.
```suggestion
				Each filter string in the [param filters] array is to be formatted similar to this: [code]*.txt;Text Files (*.txt)[/code]. See also [member FileDialog.filters].
```

Reply:
That, plus this (also applies to the addition below).
```suggestion
				Each filter string in the [param filters] array should be formatted similar to this: [code]*.txt;Text Files (*.txt)[/code]. See also [member FileDialog.filters].
```

Reply:
Shouldn't it be "similarly to this"? Or just "like this" ;)

Reply:
"like this" does sound better too. Bit nitpicky, but true.

Reply:
Filter string can also have multiple extensions, e.g `"*.txt,*.doc;Text File"`, the description part is optional (e.g, `"*.txt"` is valid).